services/alertmanager/alertmanager.go

```diff
@@ -43,20 +46,41 @@ const (
 
 	alertmanagerConfigPath     = "/etc/alertmanager.yml"
 	alertmanagerBaseConfigPath = "/srv/alertmanager/alertmanager.base.yml"
+
+	shippedRuleTemplatePath     = "/tmp/ia1/*.yml"
+	userDefinedRuleTemplatePath = "/tmp/ia2/*.yml"
+
+	envShippedRuleTemplatePath     = "PERCONA_TEST_SHIPPED_RULE_TEMPLATE_PATH"
+	envUserDefinedRuleTemplatePath = "PERCONA_TEST_USER_DEFINED_RULE_TEMPLATE_PATH"
 )
 
 // Service is responsible for interactions with Alertmanager.
 type Service struct {
-	db *reform.DB
-	l  *logrus.Entry
+	db                          *reform.DB
+	l                           *logrus.Entry
+	shippedRuleTemplatePath     string
+	userDefinedRuleTemplatePath string
+	rules                       []saas.Rule
```

Comment:
In practice, we will need to access them by names. Please change to `map[string]saas.Rule` with name in key